src/studio/src/designer/backend.Tests/Models/AltinnCoreFileTests.cs

```diff
@@ -16,13 +16,17 @@ public void CreateFromPath_ValidPath_ShouldCreateInstanse()
             var repository = "ttd-datamodels";
             var userName = "testUser";
             var repositoryRootPath = TestDataHelper.GetTestDataRepositoryDirectory(org, repository, userName);
-            var filePath = $"{repositoryRootPath}\\App\\models\\0678.xsd";
+            var fileName = "0678.xsd";
+            var directory = $"{repositoryRootPath}\\App\\models";
```

Comment:
does this test run fine on CI?
forward slash instead of escaped backslash would probably make it more comparable to how it would run in the "field" as well?

Reply:
It doesn't matter really. We use the .Net Core libraries for handling paths and they know what they are running on so slashes both ways are handled.

Reply:
OK
As long as we make sure we stick to (or support) forward-slashes when we communicate through the APIs we are good then.

Reply:
Yes. I've use forward slashes in the controller tests.